package.json

```diff
@@ -86,6 +90,12 @@
     "ts-node": "^9.0.0",
     "zip-folder": "^1.0.0"
   },
+  "husky": {
```

Comment:
Similarly to the commit-msg comment, this was recently removed. Howcome youve added it back?

Reply:
Don't know why this happened. Fixed.